src/components/ChannelsAvailabilityCard/utils.ts

```diff
@@ -0,0 +1,95 @@
+import { ChannelData } from "@saleor/channels/utils";
+import { LocalizeDate } from "@saleor/hooks/useDateLocalize";
+import { IntlShape } from "react-intl";
+
+import { Message } from "./types";
+
+export const getChannelsAvailabilityMessages = ({
+  messages,
+  channels = [],
+  intl,
+  localizeDate
+}: {
+  messages?: Message;
+  channels?: ChannelData[];
+  intl: IntlShape;
+  localizeDate: LocalizeDate;
+}): Message =>
```

Comment:
I'm sure that output of this util does not satisfy `Message` interface, more like `Record<string, Message>`

Reply:
Changes name to Messages, and it actually is 
```
export interface Messages {
  visibleLabel: string;
  hiddenLabel: string;
  visibleSecondLabel?: string;
  hiddenSecondLabel?: string;
  availableDateText?: string;
  availableLabel?: string;
  unavailableLabel?: string;
  availableSecondLabel?: string;
  setAvailabilityDateLabel?: string;
}